### PR TITLE
Asynchronous Encounters Guide

### PR DESCRIPTION
**Pull request by rahul1**
Guide on how to represent text/email based care in terms of Encounters and Communications

### CODE REVIEW

**Comment by rahul1** [comment]
fixes #1436

File: packages/docs/docs/communications/async-encounters.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+2. Create a new [Encounter](/docs/api/fhir/resources/encounter) for each patient to represent a medical encounter
+3. Set the `Encounter.subject` of each medical encounter to the corresponding patient
+4. Populate each medical encounter with the clinical details (diagnoses, reasons for visit) of the corresponding patient.
+5. Set the `Encounter.partOf` element for each medical encounter to refer to the session's [Encounter](#)
+
+The [Communication](#) resources should still be linked to the "session" encounter, but the clinical details for each patient will live on the medical encounters. You can add an additional value the `Encounter.type` element to tag encounters as either "sessions" or "medical encounters."
+
+While creating an [Encounter](/docs/api/fhir/resources/encounter) hierarchy like this is a bit more work up front, it promotes good data hygiene. Separating out medical details for each patient allows for better patient analytics and also helps with [billing](/docs/billing), as payer systems typically expect a single encounter per patient.
```
Comment (reshmakh) [inline packages/docs/docs/communications/async-encounters.md:53]: I'd add a bit stronger language here.  A well documented encounter, with the correct Practitioner, Diagnosis codes, service type is critical to correct billing.  For reference, this is a sample encounter for USCDI

```
{
  "id": "82bbbd9a-f7af-4ba3-a55d-7c4255c3bfa1",
  "meta": {
    "profile": [
      "http://hl7.org/fhir/us/core/StructureDefinition/us-core-encounter"
    ],
    "versionId": "8f8e2dc5-9a1f-4a48-91d1-da136d1f5d66",
    "lastUpdated": "2023-01-22T19:24:17.575Z",
    "author": {
      "reference": "Practitioner/ff3139bc-fe17-416b-8016-8285498e9c8c",
      "display": "Reshma Khilnani"
    },
    "project": "11ba3e91-e233-4e27-9a1c-bafdbb34061f",
    "compartment": [
      {
        "reference": "Project/11ba3e91-e233-4e27-9a1c-bafdbb34061f"
      },
      {
        "reference": "Patient/13e44a47-636b-49e2-adb3-9f19c7e0e47a",
        "display": "Mr. Dustin31 Ritchie586"
      }
    ]
  },
  "identifier": [
    {
      "use": "official",
      "system": "https://github.com/synthetichealth/synthea",
      "value": "a2590942-80e3-b27b-39ab-017a88b3fed1"
    }
  ],
  "status": "finished",
  "class": {
    "system": "http://terminology.hl7.org/CodeSystem/v3-ActCode",
    "code": "AMB"
  },
  "type": [
    {
      "coding": [
        {
          "system": "http://snomed.info/sct",
          "code": "162673000",
          "display": "General examination of patient (procedure)"
        }
      ],
      "text": "General examination of patient (procedure)"
    }
  ],
  "subject": {
    "reference": "Patient/13e44a47-636b-49e2-adb3-9f19c7e0e47a",
    "display": "Mr. Dustin31 Ritchie586"
  },
  "participant": [
    {
      "type": [
        {
          "coding": [
            {
              "system": "http://terminology.hl7.org/CodeSystem/v3-ParticipationType",
              "code": "PPRF",
              "display": "primary performer"
            }
          ],
          "text": "primary performer"
        }
      ],
      "period": {
        "start": "2020-10-15T19:33:18-04:00",
        "end": "2020-10-15T19:48:18-04:00"
      },
      "individual": {
        "reference": "Practitioner/bbb136d1-88af-4dab-b60b-beabc8ec558a",
        "display": "Dr. Blossom971 Christiansen251"
      }
    }
  ],
  "period": {
    "start": "2020-10-15T19:33:18-04:00",
    "end": "2020-10-15T19:48:18-04:00"
  },
  "location": [
    {
      "location": {
        "reference": "Location/e97cda0e-ab8b-436a-a79c-103bdc855c09",
        "display": "PCP170967"
      }
    }
  ],
  "serviceProvider": {
    "reference": "Organization/89fdb7b6-b113-4831-b382-4f07eb7222cc",
    "display": "PCP170967"
  },
  "resourceType": "Encounter"
}
```

File: packages/docs/docs/communications/async-encounters.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+To get started, you'll first need to determine what determines a "session" in your care setting.
+
+A session could be a single SMS chain, or a single email thread. Many digital health apps ask the patient to explicitly initiate a messaging session with a provider.
+
+Alternatively, if your care setting has more of an rolling interaction model (e.g. a continuous text thread), you may choose to group all communications from the same day into a session.
+
+## Representing Sessions in FHIR
+
+Each session should be represented by an [Encounter](/docs/api/fhir/resources/encounter) resource. All of the messages that are part of this session should be represented as [Communication](#) resources and can be linked to the session using the `Communication.encounter` element.
+
```
Comment (reshmakh) [inline packages/docs/docs/communications/async-encounters.md:22]: Consider an awesomic diagram here, or perhaps start with one from slides.  As this is a key point!

File: packages/docs/docs/communications/async-encounters.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+You should record the participating physicians using the `Encounter.participant` element. You can also record any family members who are part of the session here (see our [Family Relationships guide](#) ).
+
+:::tip Asynchronous Encounter Ontologies
+
+The `Encounter.class` is a required element in FHIR and should be taken from the [HL7 Act Encounter Code Valueset](https://terminology.hl7.org/3.1.0/ValueSet-v3-ActEncounterCode.html). Asynchronous care contexts will almost always use the code `VR` ("virtual").
```
Comment (reshmakh) [inline packages/docs/docs/communications/async-encounters.md:27]: This is useful.

File: packages/docs/docs/communications/async-encounters.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+To properly represent your asynchronous encounter, you should:
+
+1. Create an [Encounter](/docs/api/fhir/resources/encounter) for the session, [as described above ](#representing-sessions-in-fhir)
+2. Create a new [Encounter](/docs/api/fhir/resources/encounter) for each patient to represent a medical encounter
+3. Set the `Encounter.subject` of each medical encounter to the corresponding patient
+4. Populate each medical encounter with the clinical details (diagnoses, reasons for visit) of the corresponding patient.
+5. Set the `Encounter.partOf` element for each medical encounter to refer to the session's [Encounter](#)
+
+The [Communication](#) resources should still be linked to the "session" encounter, but the clinical details for each patient will live on the medical encounters. You can add an additional value the `Encounter.type` element to tag encounters as either "sessions" or "medical encounters."
```
Comment (reshmakh) [inline packages/docs/docs/communications/async-encounters.md:51]: # a placeholder?

**Review by reshmakh — APPROVED** [review]
Minor feedback, but this looks good.

File: packages/docs/docs/communications/async-encounters.md (hunk 3 above)
Comment (reshmakh) [reply packages/docs/docs/communications/async-encounters.md:27]: One thing that I'd add here is that may want to note where CPTs should be filled in.

Comment (reshmakh) [reply packages/docs/docs/communications/async-encounters.md:27]: May want to skim [Coded Encounters](https://app-staging.joincandidhealth.com/docs#tag/Coded-Encounters) to get a sense of which data should be included.

File: packages/docs/docs/communications/async-encounters.md (hunk 4 above)
Comment (rahul1) [reply packages/docs/docs/communications/async-encounters.md:51]: whoops!

File: packages/docs/docs/communications/async-encounters.md (hunk 3 above)
Comment (rahul1) [reply packages/docs/docs/communications/async-encounters.md:27]: Yeah, I wasn't sure how much to include in this section, as the main focus is about representing async encounters. I think. we should have a separate guide on ontologies for billing in the `Billing` section, and link from here.

File: packages/docs/docs/communications/async-encounters.md (hunk 2 above)
Comment (rahul1) [reply packages/docs/docs/communications/async-encounters.md:22]: Yeah good point

File: packages/docs/docs/communications/async-encounters.md (hunk 3 above)
Comment (reshmakh) [reply packages/docs/docs/communications/async-encounters.md:27]: Fair enough

**Pull request merged**